db/migrate/20160512012637_add_thumbnail_to_serials.rb

```diff
@@ -0,0 +1,5 @@
+class AddThumbnailToSerials < ActiveRecord::Migration
+  def change
+    add_column :serials, :thumbnail, :string, null: false
```

Comment:
Umm, I think we shouldn't delete records in migration because it may make rollback hard.
How about the following? We can use post's thumbnail for serials.

```ruby
class AddThumbnailToSerials < ActiveRecord::Migration
  def up
    add_column :serials, :thumbnail, :string

    execute <<~SQL
      UPDATE
        serials
      SET
        thumbnail = posts.thumbnail
      FROM
        posts
      WHERE
        serials.id = posts.serial_id
    SQL

    change_column_null :serials, :thumbnail, false
  end

  def down
    remove_column :serials, :thumbnail
  end
end
```